packages/web/components/complex/all-pools-table.tsx

```diff
@@ -354,8 +356,42 @@ export const AllPoolsTable: FunctionComponent<{
                 ObservablePoolWithMetric
               >
             ) => {
+              const pool = props.getValue();
+
+              const inflation = queriesCosmos.queryInflation;
+              /**
+               * If pool APR is 5 times bigger than staking APR, warn user
+               * that pool may be subject to inflation
+               */
+              const isAPRTooHigh = pool.apr
+                .toDec()
+                .gt(
+                  inflation.inflation.toDec().quo(new Dec(100)).mul(new Dec(5))
+                );
+
               return (
-                <MetricLoaderCell value={props.getValue().apr.toString()} />
+                <MetricLoaderCell
+                  isLoading={
+                    queriesOsmosis.queryIncentivizedPools.isAprFetching ||
+                    inflation.isFetching
```

Comment:
There may be a debounce issue with this, causing the animation to restart repeatedly and stutter...

Reply:
yea, adding `inflation.isFetching` caused a weird state transition. Removed it as there was no point in hiding apr while inflation loads.